test/test07.erl

```diff
@@ -0,0 +1,9 @@
+-module(test07).
```

Comment:
Would you add a testcase for negative numbers, please? :pray:

Reply:
Oops I forgot it. I'll add the testcase!

Reply:
It seems that negative big integer is compiled as a positive big integer with unary operator (`-`). 
I can't think how to test negative big integers only .beam parsing.

I created an issue https://github.com/yutopp/obeam/issues/28